Bug Reporting/ARKBugReporter.h

```diff
@@ -26,9 +26,12 @@ NS_ASSUME_NONNULL_BEGIN
 
 @protocol ARKBugReporter <NSObject>
 
-/// Called when the user has triggered the creation of a bug report.
+/// Called when the user has triggered the creation of a bug report. Logs a screenshot by default.
 - (void)composeBugReport;
 
+/// Called when the user has triggered the creation of a bug report, optionally, logs a screenshot.
+- (void)composeBugReportWithScreenshotLog:(BOOL)logScreenshot;
```

Comment:
(nit) `WithScreenshotLog` makes me expect a screenshot log object of some kind to be passed in as a parameter, when its just a boolean. `composeBugReportShouldLogScreenshot:`? Perhaps two methods hiding the `BOOL` internally, e.g. `composeBugReport`, `composeBugReportWithoutScreenshot`?